.github/workflows/ci.yml

```
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
```

This is minor but you may want to also enable cron:
```suggestion
    tags:
  schedule:
  # https://crontab.guru/#53_3_*_*_*
  - cron: 53 3 * * *  # At minute 53 past 3am every day.
```

---

Yeah, why not? Should QA job also be scheduled?

---

It could. But I guess linters are less important than legit tests.

---

Addressed in 790872f.